sdk/core/azure-core/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.11.0-beta.1 (Unreleased)
 
+### New Features
+
+- Added `BinaryData` which allows for a format agnostic representation of binary information.
```

Comment:
Can we also say about "Provides API to serialize/deserialize Object."  Since it is important feature.